.clangd

```diff
@@ -0,0 +1,5 @@
+# SPDX-FileCopyrightText: 2021-2024 The Ikarus Developers mueller@ibb.uni-stuttgart.de
+# SPDX-License-Identifier: CC0-1.0
+CompileFlags:                                         # Tweak the parse settings
+  Add: -include=build/config.h                        # Add config.h to parse first 
+  Remove: -fext-numeric-literals*                     # strip all other warning-related flags
```

Comment:
The add makes the vs code clangd extension work better I suppose? Nice!

Why is the remove statement added? Can you elaborate?

Reply:
Exactly, so with that you can tell clangd additional arguments to parse before each file, now the preprocessor constants are working correctly, e.g. the element formulations are not greyed out. 

The remove I added because cland always displayed an error at most files that a certain compiler flag is unknown to him, and so the file was always red, I don't know If you have encountered this. This seems to be a flag that is added in our cmake configuration

Reply:
![grafik](https://github.com/user-attachments/assets/8294e4f4-6597-41bf-91ba-8553a02b3911)